include/crm/pengine/internal.h

```diff
@@ -103,6 +103,11 @@ void clone_print(resource_t * rsc, const char *pre_text, long options, void *pri
 void pe__print_bundle(pe_resource_t *rsc, const char *pre_text, long options,
                       void *print_data);
 
+void native_output(resource_t * rsc, long options, void * out);
+void group_output(resource_t * rsc, long options, void * out);
+void clone_output(resource_t * rsc, long options, void * out);
+void pe__output_bundle(pe_resource_t *rsc, long options, void * out);
```

Comment:
Since these are new functions, we can follow the new style for all of them: pe__output_TYPE(pe_resource_t ...)

The "pe" prefix helps avoid naming conflicts with (theoretical) user functions; the double underbar indicates that it's not part of the public API; and pe_resource_t is clearer than resource_t since we also have crm_resource_t in the controller.

Reply:
Done.